### PR TITLE
wd_core.au3 _WD_ExecuteScript() $bReturnAsValue

### PR DESCRIPTION
**Pull request by mlipok**
​I was thinking about adding this `_WD_GetDevicePixelRatio()` to `wd_helper.au3` as there is similar `_WD_IsWindowTop()` which I usesd as an emplate for my _WD_GetDevicePixelRatio()

but...
Maybe we can add a new parameter $bReturnAsValue to `_WD_ExecuteScript`

​https://www.autoitscript.com/forum/topic/205553-webdriver-udf-help-support-iii/?do=findComment&comment=1493911

### CODE REVIEW

File: wd_core.au3 [hunk 1]
```diff
@@ -843,30 +843,32 @@ EndFunc   ;==>_WD_ElementAction
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_ExecuteScript
 ; Description ...: Execute Javascipt commands.
-; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default]])
-; Parameters ....: $sSession   - Session ID from _WD_CreateSession
-;                  $sScript    - Javascript command(s) to run
-;                  $sArguments - [optional] String of arguments in JSON format
+; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default[, $bReturnAsValue = False]]])
+; Parameters ....: $sSession            - Session ID from _WD_CreateSession
```
Comment (Danp2) [inline wd_core.au3]: I think these extra spaces will cause issues with building the help file

File: wd_core.au3 [hunk 2]
```diff
@@ -885,6 +887,12 @@ Func _WD_ExecuteScript($sSession, $sScript, $sArguments = Default, $bAsync = Def
 		Return SetError(__WD_Error($sFuncName, $iErr, "HTTP status = " & $_WD_HTTPRESULT), $_WD_HTTPRESULT, $sResponse)
 	EndIf
 
+	If $bReturnAsValue Then
+		Local $oJSON = Json_Decode($sResponse)
+		Local $vValue = Json_Get($oJSON, "[value]")
+		Return SetError($_WD_ERROR_Success, 0, $vValue)
```
Comment (Danp2) [inline wd_core.au3:893]: I would prefer to keep a single point of return. Should be fairly simple to accomplish.

**Review by Danp2 — CHANGES_REQUESTED** [review]

File: wd_core.au3 (hunk 1 above)
Comment (mlipok) [reply wd_core.au3]: looking into:
https://github.com/Danp2/WebDriver/pull/110/files
all is fine

File: wd_core.au3 (hunk 2 above)
Comment (mlipok) [reply wd_core.au3]: Hope third commit is fine.

**Comment by mlipok** [comment]
Just checking rerquest feature.
Should I use them when each time I submit my commits ?

**Review by Danp2 — CHANGES_REQUESTED** [review]

File: wd_core.au3 [hunk 3]
```diff
@@ -843,30 +843,32 @@ EndFunc   ;==>_WD_ElementAction
...
-; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default]])
-; Parameters ....: $sSession   - Session ID from _WD_CreateSession
-;                  $sScript    - Javascript command(s) to run
-;                  $sArguments - [optional] String of arguments in JSON format
+; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default[, $bReturnAsValue = False]]])
+; Parameters ....: $sSession            - Session ID from _WD_CreateSession
+;                  $sScript             - Javascript command(s) to run
+;                  $sArguments          - [optional] String of arguments in JSON format
 ;                  $bAsync     - [optional] Perform request asyncronously? Default is False
+;                  $bReturnAsValue      - [optional] Return Value instead WebDriver Resopnse ? Default is False.
```
[the remark this answers is not in the document]
Comment (Danp2) [reply wd_core.au3:851]: > Just checking rerquest feature. Should I use them when each time I submit my commits ?

I assume you mean the Request Review feature. It's fine to use it.

**Review by Danp2 — APPROVED** [review]

**Pull request merged**